edx_notifications/server/api/consumer.py

```diff
@@ -183,9 +187,28 @@ def post(self, request):
             return JsonResponse({"message": "Invalid notification payload"}, status=status.HTTP_400_BAD_REQUEST)
 
         user_ids = User.objects.filter(username__in=usernames).values_list('id', flat=True)
+        community_url = notification_data['categoryData']['slug']
```

Comment:
Can we write it as notification_data.get('categoryData', {}).get('slug', '') ?

Reply:
Yes we can.